libs/src/services/solanaWeb3Manager/rewardsAttester.js

```diff
@@ -674,9 +674,7 @@ class RewardsAttester {
         noRetry.push(res)
         const errorType = {
           [errors.HCAPTCHA]: 'hcaptcha',
-          [errors.COGNITO_FLOW]: 'cognito',
-          [errors.BLOCKED]: 'blocked',
-          [errors.OTHER]: 'other'
```

Comment:
Currently in progress of converting this file to TS – I can include this in my PR rather than include it here